CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 ## Unreleased
 
+### Upgrade notes
+
+#### Deduplicating endorsements
+
+We have identified a case when the same user can endorse the same resource multiple times. This is a bug that we have fixed in this release, but we need to clean up the existing duplicated endorsements. We have added a new task that helps you clean the duplicated endorsements.
+
+```bash
+bundle exec rails decidim:upgrade:fix_duplicate_endorsements
+```
+
+You can see more details about this change on PR [\#11853](https://github.com/decidim/decidim/pull/11853)
```

Comment:
I am not sure if here we should leave old PR, or new one.. .

Reply:
Seeing how it was done for other PRs in the changelog, we leave out the original PR as that's the one giving the information. 
Mind that this PR will appear in the PR when we create the changelog with the `bin/changelog_generator` script